app/src/main/java/com/github/orkest/data/SpotifyMultimedia.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.orkest.data
+
+import android.util.Log
+import com.github.orkest.ui.sharing.SharingComposeActivity
+import okhttp3.*
+import org.json.JSONObject
+import java.io.IOException
+import java.util.concurrent.CompletableFuture
+
+/*
+    This class is used to provide an interface to
+    access spotify multimedia data (images, videos, etc)
+    from the spotify API.
+ */
+class SpotifyMultimedia {
+
+    // This function is used to get the album cover image
+    fun getAlbumCoverImageUrl(albumId: String, accessToken: String): CompletableFuture<String> {
+
+        val future = CompletableFuture<String>()
+        val client = OkHttpClient()
+
```

Comment:
The OkHttpClient is created everytime the function is called. Maybe it would be better to create a single instance and reuse it across multiple calls to make the code more efficient